build/pipelines/templates/build-console-steps.yml

```diff
@@ -36,16 +36,27 @@ steps:
     restoreDirectory: '$(Build.SourcesDirectory)/packages'
 
 - task: VSBuild@1
-  displayName: 'Build solution **\OpenConsole.sln'
+  displayName: 'Build solution **\OpenConsole.sln (no packages)'
   inputs:
     solution: '**\OpenConsole.sln'
     vsVersion: 16.0
     platform: '$(BuildPlatform)'
     configuration: '$(BuildConfiguration)'
-    msbuildArgs: ${{ parameters.additionalBuildArguments }}
+    msbuildArgs: "${{ parameters.additionalBuildArguments }} /p:BuildingInsideVisualStudio=false;_WapBuildingInsideVisualStudio=false;GenerateAppxPackageOnBuild=false"

```

Comment:
Ehhhhh I thought `BuildingInsideVisualStudio` was magic and set correctly. Why did this have to be overriden? Can we leave comments in the YMLs?

Reply:
I'll comment (and discuss) this in the PR I split off this one to fix the build.